.github/workflows/build-and-test.yml

```
@@ -1,25 +1,52 @@
 jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+
+    steps: 
+      - name: Run tests
+        run: |
+          cd frontend
+          pnpm lint
+
+  test: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)